src/index.mjs

```diff
@@ -27,18 +27,17 @@ export class Counter {
     // Apply requested action.
     let url = new URL(request.url);
 
-    // explanation of why this is safe and fast:
-    // https://blog.cloudflare.com/durable-objects-easy-fast-correct-choose-three/
+    // Durable Object storage is automatically cached in-memory, so reading the
+    // same key every request is fast. (That said, you could also store the
+    // value in a class member if you prefer.)
```

Comment:
If someone stores the value in a class member, are they still going to need blockConcurrencyWhile in the constructor as in the previous version of the template?

Reply:
They could avoid it by relying on input gates, but if I were writing example code I'd probably use blockConcurrencyWhile, yes.

Reply:
Because the constructor isn't async, is there another way to do it?

Reason I bring it up is I think the counter is our only example in the docs of actually using blockConcurrencyWhile, I guess I can move it from https://developers.cloudflare.com/workers/learning/using-durable-objects#example---counter to  https://developers.cloudflare.com/workers/runtime-apis/durable-objects#durable-object-class-definition section

Reply:
It's true the constructor technically isn't an async function, but input locks still apply if you use `.then()` syntax. It would definitely be confusing to rely on input locks in the constructor, though, so I wouldn't recommend it for an example.